drift/test/generated/todos.dart

```diff
@@ -288,6 +288,24 @@ const uuidType = DialectAwareSqlType<UuidValue>.via(
   overrides: {SqlDialect.postgres: NativeUuidType()},
 );
 
+/// Tables to test that foreign keys which target foreign keys
+/// columns are handled correctly in manager code
+class FkToPk0 extends Table {
```

Comment:
I prefer these as a unit test somewhere in `drift_dev` that asserts that the generated code doesn't include the relevant snippet (using `emulateDriftBuild` + `decodedMatches(isNot(contains(...)))`.

Reply:
Done